src/explore-education-statistics-common/src/modules/table-tool/components/MultiHeaderTable.module.scss

```diff
@@ -47,34 +47,39 @@ $border-width: 1px;
 
 .tableHead {
   td {
-    background: #fff;
+    background: govuk-colour('white');
     position: relative;
     z-index: 3;
   }
 
-  .emptyColumnHeaderCell {
-    background: govuk-colour('light-grey');
-    height: 46px;
-    z-index: 1;
+  // Uses the spacer cell to cover the left border
+  // of header cells. We can't just remove the
+  // left border on the first cell in header rows
+  // because this causes inconsistent styling when there
+  // are merged cells which span down through rows.
+  td::after {
```

Comment:
Not a problem if you prefer it like this, but just wondering if having a left border is really a problem. Part of me wonders if we could just leave it in rather than do this?

Reply:
Not sure it looks quite right with a left border:
![left-border](https://github.com/dfe-analytical-services/explore-education-statistics/assets/81572860/746a8b7b-ecb6-4b39-b147-c81f372ff043)
Maybe if had a right one as well:
![borders](https://github.com/dfe-analytical-services/explore-education-statistics/assets/81572860/c3201183-b9b9-4700-aa14-ffbf49417ac7)

Either way, we should check with Cam before making a style change on the pubic site, so for now I'd prefer to keep the style as is.

Reply:
I was also wondering if it'd look better if we just shaded all of `thead` as well to make it look more symmetrical.

Anyway, fine to keep it as-is for now.